config/initializers/decidim.rb

```diff
@@ -12,8 +12,7 @@
   # Geocoder configuration
   config.geocoder = {
     static_map_url: "https://image.maps.cit.api.here.com/mia/1.6/mapview",
-    here_app_id: Rails.application.secrets.geocoder[:here_app_id],
-    here_app_code: Rails.application.secrets.geocoder[:here_app_code]
+    here_api_key: Rails.application.secrets.geocoder[:here_api_key]
```

Comment:
we should set the `here_api_key` in the `config/secrets.yml` file also

Reply:
Done!